<?xml version="1.0"?>

<launch>

  <!-- Start f/t sensor -->
  <include file="$(find virtual_ft_sensor)/launch/sensor.launch" />

  <arg name="robot" default="lwr4p"/>

  <include file="$(find robot_sim)/launch/lwr4p_bh282.launch">
    <arg name="robot_config" value="$(find robot_sim)/config/$(arg robot).yaml" />
  </include>

  <!-- Object load controller -->
  <arg name="node_name" default="dmp_join_node" />
  <node pkg="dmp_join" type="dmp_join" name="$(arg node_name)"
        output="screen" cwd="node">

        <param name="robot_type" value="robot_sim" type="str"/>

  </node>

</launch>
